src/Storefront/Controller/UxComponentLibraryController.php

```diff
@@ -25,6 +25,10 @@ public function index(): Response
     {
         $components = $this->uxComponentHelper->getComponents(true, true);
 
+        dump($components);
+        die();
```

Comment:
```suggestion
        dd($components);
```

does the same :wink: